packages/terra-site/src/examples/form/examples/Input.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Input from 'terra-form/lib/components/Input';
+
+const InputExamples = () => (
+  <form>
+    <Input
+      name="foo"
+      defaultValue="bar"
+    />
+  </form>
+);
+
+export default InputExamples;
```

Comment:
My plan is to add a form example at the bottom of the page to handle that for all components. How does that sound?